packages/amplify-util-uibuilder/package.json

```diff
@@ -12,11 +12,41 @@
     "access": "public"
   },
   "dependencies": {
-    "@aws-amplify/codegen-ui": "^1.0.0",
-    "@aws-amplify/codegen-ui-react": "^1.0.0",
+    "@aws-amplify/codegen-ui": "1.1.0",
+    "@aws-amplify/codegen-ui-react": "1.1.0",
```

Comment:
This should now point to 1.2.0, which is the version including the bug fixes which we've pushed Today.

Reply:
Should these versions have a `^` prefix so that we don't need a manual update for each release?

Reply:
Given the fragility we've seen with the packaged cli and codegen transpilation we're making the explicit decision to pin versions until post-GA for studio. We want to ensure we can smoke test every uibuilder-related CLI release before it goes.

For context, before launch we had an emergent bug where invoking the typescript transpile failed due to the way `pkg` intercepts fsCopy commands, and we were using a `temp` directory. We've since moved to a virtual filesystem for generating types in codegen to mitigate this issue, and this PR is re-enabling that functionality (available in 1.2.0 and up).